Relationships/RelationshipResolver.cs

```diff
@@ -280,5 +296,130 @@ public List<CkanModule> ModList()
             var modules = new HashSet<CkanModule>(modlist.Values);
             return modules.ToList();
         }
+
+        /// <summary>
+        ///  Returns a IList consisting of keyValuePairs containing conflicting mods.
+        /// Note: (a,b) in the list should imply that (b,a) is in the list. 
+        /// </summary>
+        public Dictionary<Module, String> ConflictList
+        {
+            get
+            {
+                var dict = new Dictionary<Module, String>();
+                foreach (var conflict in conflicts)
+                {
+                    var module = conflict.Key;
+                    dict[module] = string.Format("{0} conflicts with {1}\n\n{0}:\n{2}\n{1}:\n{3}",
+                        module.identifier, conflict.Value.identifier,
+                        ReasonStringFor(module), ReasonStringFor(conflict.Value));
+                    ;
+                }
+                return dict;
+            }
+        }
+
+        public bool IsConsistant => conflicts.Any();
```

Comment:
@RichardLake : This line makes xbuild on my machine cry. I suspect it's because it's an older version of mono, as jenkins thinks it's fine.